website/docs/sql_ddl.md

```diff
@@ -0,0 +1,408 @@
+---
+title: SQL DDL
+summary: "In this page, we introduce how to create tables with Hudi."
```

Comment:
this page is specific to Spark right? it may be confusing to readers if we don't qualify all that.